public/app/features/dashboard-scene/panel-edit/PanelDataPane/DataProviderSharer.tsx

```diff
@@ -0,0 +1,48 @@
+import { Observable } from 'rxjs';
+
+import {
+  SceneDataProvider,
+  SceneDataProviderResult,
+  SceneDataState,
+  SceneObjectBase,
+  SceneObjectRef,
+} from '@grafana/scenes';
+
+export interface DataProviderSharerState extends SceneDataState {
+  source: SceneObjectRef<SceneDataProvider>;
+}
+
+export class DataProviderSharer extends SceneObjectBase<DataProviderSharerState> implements SceneDataProvider {
```

Comment:
this is a util data provider (that we maybe should add to scenes lib), that allow one scene object to share the data from another. Used by the table view to share the data from the original panel